files/deploy-amlarc.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+subscription_id="<YOUR_SUBSCRIPTION_ID>"
+resource_group="<YOUR_RESOURCE_GROUP>"
+cluster_name="<YOUR_AKS_CLUSTER_NAME>"
+
+arcml_extension_name="arcml-extension"
+
+ssl_cname="<YOUR_SSL_CNAME>"
+ssl_cert_pem_file="<YOUR_CERT_PEM_FILE>"
+ssl_key_pem_file="<YOUR_KEY_PEM_FILE>"
+
+# STEP1 Register feature providers
+echo 'Register features...'
+az feature register --namespace Microsoft.ContainerService -n AKS-ExtensionManager --subscription "$subscription_id"> /dev/null
```

Comment:
May the output help debug? I see you ignore it

Reply:
Print outputs to help debug.